capacitor.config.ts

```diff
@@ -0,0 +1,16 @@
+import { CapacitorConfig } from '@capacitor/cli';
```

Comment:
Do we need a capacitor configuration file here?

Reply:
yes it is necessary!